backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/Preloader.java

```diff
@@ -117,8 +119,8 @@ public Preloader (String newBaseURL) {
 
 	public void preload (final String assetFileUrl, final PreloaderCallback callback) {
 		final AssetDownloader loader = new AssetDownloader();
-		
-		loader.loadText(baseUrl + assetFileUrl, new AssetLoaderListener<String>() {
+
+		loader.loadText(baseUrl + assetFileUrl + "?etag=" + System.currentTimeMillis(), new AssetLoaderListener<String>() {
```

Comment:
This looks like an approach to disable the cache completely. If used, md5 hash is not needed any more. 
I see the advantage of a md5 hash that the cache _can_ be used.

Reply:
Okay, I know better now. This is for the assets.txt. And needed.